scripts/utils/update_package.py

```diff
@@ -93,15 +114,68 @@ def update_github_url(package):
         content = content.replace(sha256, latest_sha256).replace(sha256.upper(), latest_sha256)
 
     content = content.replace(version, latest_version)
-    with open(chocolateyinstall_path, "w") as file:
+    with open(install_script_path, "w") as file:
         file.write(content)
 
-    nuspec_path = f"packages/{package}/{package}.nuspec"
-    with open(nuspec_path, "r") as file:
-        content = file.read()
-    latest_version, content = replace_version(latest_version, content)
-    with open(nuspec_path, "w") as file:
+    update_nuspec_version(package, latest_version)
+
+    return latest_version
+
+
+def get_increased_version(url, version):
+    version_list_original = version.split(".")
+    # Try all possible increased versions, for example for 12.0.1
+    # ['13.0.1', '12.1.1', '12.0.2', '12.0.1.1']
+    versions = []
+    for i in range(len(version_list_original)):
+        version_list = version_list_original.copy()
+        version_list[i] = str(int(version_list[i]) + 1)
+        versions.append(".".join(version_list))
+    # New possible segment
+    versions.append(version + ".1")
```

Comment:
what about .0, .2, .3, etc., i.e., does it suffice to assume it would be always .1?

Reply:
There was a bug in this code, I have just updated it. I am assuming there will always be an update by 1. So for example for `12.0.1`, I am trying the possible version updates: `'12.0.1.1', '13', '13.0', '13.0.0', '13.0.0.0', '12.1', '12.1.0', '12.0.2'`. If a tool is updated twice, it will take several iterations to update to the newest version. But as we update every week, I think this is fine.